site/sfguides/src/data_mapping_in_native_apps/data_mapping_in_native_apps.md

```diff
@@ -60,27 +61,58 @@ The application provides a user interface which allows the consumer of the appli
 
+> aside positive
+> 
+> **Note** - The following steps explain how to create the app using the Snowflake CLI, as this option presents as a faster, more straightforward way to build Native Apps. You can also accomplish it by manually creating the folder structure and executing the SQL commands in Snowsight, but this alternative is out of the scope for this Quickstart.
+
+The files explained in the following steps, are already present in the project folder, and it is merely a deeper explanation of its contents.
+
+
+<!-- ------------------------ -->
+## Snowflake CLI First Steps
+Duration: 7
+
+The datamapping native app project resides in a Snowflake repository which contains several app examples for the user to learn its way into Snowflake Native Apps.
+
+The instructions on cloning the project repository and its execution are given in the readme file of this github [link NEEDS UPDATE AFTER PR](https://github.com/snowflakedb/native-apps-examples).
+
+### Clone instructions 
+
+1. Clone the project repo in your local machine. Repo could be cloned running:
+    ```sh
+    snow app init --template-repo git@github.com:snowflakedb/native-apps-examples.git --template datamapping-app
+    ```
+2. Replace the CSV filepath inside the **prepare_data.sh** file, line 28, like this:  
+    ```sh
+    snow object stage copy /USER_PATH_HERE/IP2LOCATION-LITE-DB11.CSV @location_data_stage --database ip2location --schema ip2location
+    ```
+
 <!-- ------------------------ -->
 ## Building the Application
-Duration: 10
+Duration: 7
 
-The application itself is a simple one and has been broken down into three parts.
+The application itself has been broken down into three parts.
 
 * The building of the application package on the provider and the sharing of the lookup database with the application. 
 * The building of the application install script which contains the functionality to accept an IP address, look it up in the database we just shared with the application and write back enhanced data from the application. 
 * Arguably (certainly for this Quickstart) the most important part which is the user interface written using Streamlit. This is where we will do the mappings.
 
 To do the enhancement of the IP addresses we will use a dataset called DB11 from [IP2LOCATION](https://www.ip2location.com/database/ip2location). There is a free version of this database available [here](https://lite.ip2location.com/database/db11-ip-country-region-city-latitude-longitude-zipcode-timezone), which is the one we will use in this quickstart.  If you do not have an account with them already you will need to create one. Download the dataset as a CSV file so it is ready to import  into the provider account.
 
-Head over to Snowsight on the provider account and open a new worksheet.
-
 The first thing you need to do is create a new database which will serve as the lookup database for the application
```

Comment:
We should add a period to this sentence. Also, the below code block is doing much more than just creating a database; is this all part of the prepare_data.sh file that's been copied in?